docs/topics/third-party.rst

```diff
@@ -12,6 +12,8 @@ If you would like to run your own Add-ons server you may want to update addons-s
 custom templates and move the Firefox Accounts management to a `django authentication backend`_.
 
 Another option would be to add any APIs that you required and write a custom frontend. This work is
-already underway and should be completed at some point but help is always welcome.
+already underway and should be completed at some point but help is always welcome. You can find
+the API work in this project and the fronend work in `addons-frontend`_.
```

Comment:
Fixed in https://github.com/mozilla/addons-server/commit/2b8fa029643b346b5d0719376cb1886d68c41017